modules/upgrade/pages/migrate/data-migration.adoc

```diff
@@ -2,9 +2,7 @@
 :description: Migrate external data to Redpanda with MirrorMaker 2.
```

Comment:
```suggestion
:description: Migrate data from Kafka or replicate data between Redpanda clusters with MirrorMaker 2.
```

Reply:
@kbatuigas this description only appears on the index page--it doesn't get duplicated on data-migration.adoc. So this edit only updates https://deploy-preview-58--redpanda-docs-preview.netlify.app/current/upgrade/migrate/